extensions/guacamole-vault/modules/guacamole-vault-ksm/src/main/resources/translations/en.json

```diff
@@ -6,7 +6,10 @@
 
     "CONNECTION_GROUP_ATTRIBUTES" : {
         "SECTION_HEADER_KSM_CONFIG" : "Keeper Secrets Manager",
-        "FIELD_HEADER_KSM_CONFIG"   : "KSM Service Configuration "
+        "FIELD_HEADER_KSM_CONFIG"   : "KSM Service Configuration ",
+
+        "ERROR_INVALID_KSM_CONFIG_BLOB": "Invalid KSM config blob provided",
+        "ERROR_INVALID_KSM_ONE_TIME_TOKEN": "Invalid KSM one-time token provided"
```

Comment:
I think we should make these a bit more friendly and helpful, so that the admin has a good idea what has failed and how to resolve. ie: "The provided base64-encoded KSM configuration blob is not valid. Please verify that you have copied the entire blob." or "The provided configuration is not a valid KSM one-time token or base64-encoded configuration blob. Please verify ... etc."

Reply:
Sure.

Reply:
How do these look?

Reply:
These look much better.